src/Catty/Helpers/Util/Util.swift

```diff
@@ -280,4 +280,33 @@ func synchronized(lock: AnyObject, closure: () -> Void) {
 
         return nil
     }
+
+    class func getRecentlyUsedBricks() -> [String] {
```

Comment:
To avoid that the `Util` is becoming too large, it is recommended to create a new class for your feature (e.g., `RecentlyUsedBricksManager` in the same folder as the `BrickManager` plus a separate test).